pyproject.toml

```diff
@@ -68,6 +68,7 @@ version_files = [
   "crates/uv/Cargo.toml",
   "crates/uv-version/Cargo.toml",
   "docs/guides/integration/pre-commit.md",
+  "docs/guides/integration/github.md",
```

Comment:
Not sure of the heuristic used by `rooster` to find references to uv versions, so not sure if this will actually work.

Reply:
Yeah this will work, it replaces the last version number with the new version number — that's all. Thank you!